parsers/miseq/parser.py

```diff
@@ -10,9 +10,19 @@
 
 class Parser:
 
+    SAMPLE_SHEET_FILE_NAME = 'SampleSheet.csv'
+    UPLOAD_COMPLETE_FILE_NAME = 'CompletedJobInfo.xml'
+
     @staticmethod
-    def get_sample_sheet_file_name():
-        return 'SampleSheet.csv'
+    def get_required_file_list():
```

Comment:
Adding in the `CompletedJobInfo.xml` requirement.

Reply:
Do we *need* the `CompletedJobInfo.xml` file to upload?  Or is it only for automated uploads?  If it's only for auto uploads does it need to block uploading for manual uploads?